src/layouts/Layout.astro

```diff
@@ -13,12 +32,12 @@ const { title } = Astro.props;
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width" />
     <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
-    <title>{title ?? "Agency Homepage"}</title>
+    <title>{collection.agencyName || "Agency Homepage"}</title>
```

Comment:
not a big deal but typically you want the page title, then a suffix referring to the agency/website name. So homepage first.